tools/clang/test/HLSLFileCheck/disassembler/geometrytest.hlsl

```diff
@@ -0,0 +1,23 @@
+// RUN: %dxc -E main -T gs_6_0 %s | FileCheck %s
+
+// Make sure load input has precise.
+// CHECK: OutputTopology=point
+
+struct MyStruct
+{
+  precise  float4 pos : SV_Position;
+};
+
+
+[maxvertexcount(12)]
+void main(point float4 array[1] : COORD, inout PointStream<MyStruct> OutputStream0)
+{
+  float4 r = array[0];
+  MyStruct a = (MyStruct)0;
+
+  a.pos = array[r.x] + 1;
+
+  OutputStream0.Append(a);
+  OutputStream0.RestartStrip();
+
+}
```

Comment:
This is now a duplicate of gs_precise_output.hlsl. We could have a dedicated test that verifies all the GS pipeline runtime info. I think testing the full list and using different explicit topologies to verify that they are printed properly would be useful. If you were to convert this shader into that test, that would be alright.